addons/escoria-core/game/scenes/inventory/inventory_ui.gd

```diff
@@ -46,23 +46,30 @@ func add_new_item_by_id(item_id : String) -> void:
 		item_id = item_id.rsplit("i/", false)[0]
 	if not items_ids_in_inventory.has(item_id):
 		if not escoria.object_manager.has(item_id):
-			escoria.logger.report_errors(
-				"inventory_ui.gd:add_new_item_by_id()",
-				[
-					"Item global id '%s' does not exist." % item_id, 
-					"Check item's id in ESCORIA_ALL_ITEMS scene."
-				]
-			)
-		if not all_items.get_inventory_item(item_id):
-			escoria.logger.report_errors(
-				"inventory_ui.gd:add_new_item_by_id()",
-				[
-					"Item global id '%s' doesn't have a " +\
-							"corresponding inventory item." % item_id, 
-					"Check item's id in ESCORIA_ALL_ITEMS scene."
-				]
-			)
-		var item_inventory_button = all_items.get_inventory_item(item_id).duplicate()
+			var inventory_file = "%s%s.tscn" % [
```

Comment:
I often run into issues if the path doesn't include the trailing /.
Is there a solution to ensure its presence?

Reply:
Oh, yes. Let's use trim_suffix("/") on each directory and add a /. This will remove the trailing / if it exists or do nothing if not.

Reply:
I changed it here and on the command_directories setting as well. Good idea!